modules/microshift-firewall-update-for-service.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-firewall.adoc
+
+
+:_content-type: CONCEPT
+[id="microshift-firewall-update-for-service_{context}"]
+= Updating firewall ports when a service is exposed
+
+Firewalld is often active when you run services on {microshift-short}. This can disrupt certain services on {microshift-short} because traffic to the ports might be blocked by the firewall. You need to ensure the necessary firewall ports are open if you want certain services to be accessible from outside the host. There are several options for opening your ports:
+
+* Services of the `NodePort` and `LoadBalancer` type are automatically available with OVN-K.
++
+In these cases, OVN-K adds iptables rules so the traffic to the node IP address is delivered to the relevant ports. This is done using the PREROUTING rule chain and is then forwarded to the OVN-K to bypass the firewalld rules for local host ports and services. Iptables and firewalld are backed by nftables in RHEL9. The nftables rules, that the iptables generates, always has priority over the rules that the firewalld generates.
+
+* Pods with the `HostPort` parameter settings are automatically available. This also includes the router-default pod, which uses ports 80 and 443.
```

Comment:
* Same confusion here. What about ports that don't have this parameter set?
* Should `router-default` be in backticks?

Reply:
I still think this concept is meant to be for certain cases, but will also check